python/turbine_models/custom_models/llm_runner.py

```diff
@@ -46,42 +55,151 @@
 """,
     help="prompt for llm model",
 )
+parser.add_argument(
+    "--chat_mode",
+    action="store_true",
+    help="Runs both turbine vmfb and a torch model to compare results",
```

Comment:
Nit: Is this the correct help description?

Reply:
good catch!

Reply:
fixed!